### DISCUSSION TITLE
Welcome to HAL Robotics

### DISCUSSION THREAD

**Discussion by sebandraos** [HALRobotics/Beta#81] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you have an issue, please look through the [issues](https://github.com/HALRobotics/Beta/issues?q=is%3Aissue), both open and closed, to see if your issue has been previously solved. If it hasn't, feel free to open a new issue and provide the information requested by the template.

**Comment by keshang-ctrl** [comment]
Keep Getting this error
HAL: Method not found: 'Unity.IUnityContainer Prism.Unity.Ioc.UnityContainerExtension.get_Instance()'.

Hal wont show up in grasshopper

**Reply by sebandraos** [reply]
Hi @keshang-ctrl ,
This sounds like a proper issue. It actually sounds like [this issue](https://github.com/HALRobotics/Beta/issues/77). Please have a read of those comments and provide the information that I asked for there and I can reopen that issue for tracking.
Thanks

**Reply by keshang-ctrl** [reply]
I've provided the information in that thread
